app/src/components/NotificationDropdown.vue

```diff
@@ -75,18 +76,32 @@ const {
 watch(notifications, () => {
   totalPages.value = Math.ceil(notifications.value.data.length / itemsPerPage.value)
 })
+
 const isUnread = computed(() => {
   const idx = notifications.value?.data.findIndex(
     (notification: any) => notification.isRead === false
   )
   return idx > -1
 })
 
-const updateNotification = async (id: number | string | null) => {
-  updateEndPoint.value = `notification/${id}`
+const isInvitation = (notification: Notification) => {
+  if (notification.resource) {
+    const resourceArr = notification.resource.split('/')
+    if (resourceArr[0] === 'teams') return true
+  }
+
+  return false
+}
+
+const updateNotification = async (notification: Notification) => {
+  updateEndPoint.value = `notification/${notification.id}`
 
   await executeUpdateNotifications()
   await executeFetchNotifications()
+
+  if (isInvitation(notification) && notification.resource) {
+    router.push(`/${notification.resource}`)
```

Comment:
The test errors have got to do something with the router being called from the components (MetaMask provider is somehow not accessible from the component level I think). 
Comment this, and you'll be able to see that the tests pass.

Reply:
The tests pass but I lose the functionality

Reply:
Yeah I get that, I was just pointing the source of the error.